src/components/pages/order/main/Product.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { fakeMenu2 } from "../../../../fakeData/fakeMenu";
+import { formatPrice } from "../../../../utils/math";
+import Card from "../../../reusable-ui/Card";
+
+export default function Product() {
```

Comment:
```suggestion
export default function Products() {
```

car ici, tu n'as pas qu'un seul produit mais une "liste" de produits.